e2e_tests/tests/per_provider/normal_tests/asym_sign_verify.rs

```diff
@@ -207,9 +258,62 @@ fn sign_hash_not_permitted() -> Result<()> {
     Ok(())
 }
 
+#[cfg(any(feature = "mbed-crypto-provider", feature = "cryptoauthlib-provider"))]
+#[test]
+fn sign_hash_not_permitted_ecc() -> Result<()> {
```

Comment:
I think a few of these could/should be set to run against other providers too, but it's something for us to figure out in the future. Thanks for adding the tests!